public/main/group/member_settings.php

```diff
@@ -227,6 +230,25 @@ function check_group_members($value)
 
 $form->setDefaults($defaults);
 echo GroupManager::getSettingBar('member');
-$form->display();
+
+// check if group has a CGroupRelUsergroup
+$courseInfo = api_get_course_info_by_id(api_get_course_int_id());
+
+if (GroupManager::is_group_linked_to_usergroup($groupEntity)) {
+
+    echo '<div class="alert alert-info">Ce groupe est lié à la classe ' .$courseInfo['title'].'. La liste de ses membres dépend des membres de la classe et ne peut-être modifiée.
```

Comment:
This is in French. Has to be translated to English, and because it includes a variable (`$courseInfo['title']), it should be a string with a `%s` in it, and the get_lang() call should be called inside a sprintf() call. Look for sprintf in the code, I'm sure you'll find many examples.